cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/events/WorkaroundMicroTime.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.operator.resource.events;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import io.fabric8.kubernetes.api.model.MicroTime;
+
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Works around an issue in fabric8 where MicroTime serialises incorrectly and is rejected by the API server
+ * I'm not sure if it's overly precise in how it formats timezones and microseconds for the Go server consuming this code,
+ *
+ * @see <a href="https://github.com/fabric8io/kubernetes-client/issues/3240">Relevant issue</a>
+ */
```

Comment:
Can you please make sure we have this covered also in a Strimzi issue? (you can open it only when this gets merged, no need to do it right now) That way, it will be more visiable than just as a comment in the code.

Reply:
Will do :+1:

Reply:
Have removed this now, as Fabric8 issue has been fixed.